### ISSUE TITLE
a11y: IconButton with Indicator relies on colour to convey meaning

### ISSUE THREAD

**Issue by t3chguy** [element-hq/compound#341]
# Platform

- Web

# Description

https://compound.element.io/?path=/story/compound-web_button-iconbutton--with-indicator&args=indicator:critical uses colour alone to differentiate the indicator between default/success/critical - this is not accessible

Ref https://element-io.atlassian.net/browse/PSB-968
